msom/src/main/java/pl/edu/uj/fais/wpz/msom/web/helpers/ModuleForm.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package pl.edu.uj.fais.wpz.msom.web.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+import pl.edu.uj.fais.wpz.msom.entities.ControllerUnit;
+
+/**
+ *
+ * @author debra
+ */
+public class ModuleForm {
```

Comment:
wlasnie dlatego tyle zeszlo bo probowalam ale nie da sie to tak latwo zrobic jak np zrobiles przy task gdzie miales pole typu tasktype. wiec musialam zrobic maly workaround.